nginx-fem-redirects.conf

```diff
@@ -1,4 +1,28 @@
 set $fe_project_uri "https://fe-project.zooniverse.org";
+set $fe_content_pages_uri "https://fe-content-pages.zooniverse.org";
+
+# Project app data and static files
+location ~* ^/projects/(?:_next|assets)/.+?$ {
+    resolver 1.1.1.1;
+    proxy_pass $fe_project_uri;
+    proxy_set_header Host fe-project.zooniverse.org;
```

Comment:
can we extract these host headers to variables as well?

Reply:
Good idea. 👍🏻